templates/item/annotation.macros.j2

```diff
@@ -92,6 +97,58 @@
 {% endmacro %}
 
 
+{# @output annotation #}
+{% macro output(data) %}
+  {% if data %}
+    {{ param_list(
+      name='{CSS output}',
+      value_type='code block',
+      description=data
+      ) }}
+  {% endif %}
+{% endmacro %}
+
+
+{# @content annotation #}
+{% macro content(data) %}
+  {% if data %}
+    {{ param_list(
+      name='@content',
+      value_type='code block',
+      description=data
+      ) }}
+  {% endif %}
+{% endmacro %}
+
+
+{# @throw annotation #}
+{% macro throw(data) %}
+  {% if (data.length > 0) %}
+    {% for item in data %}
+      {{ param_list(
+        name='@error',
+        description=item
+        ) }}
+    {% endfor %}
+  {% endif %}
+{% endmacro %}
+
+
+{# Group require and used-by in a single block #}
+{% macro require_or_used(item, display_private=false) %}
+  {# Add a wrapper if we have both #}
```

Comment:
I like the look of these side by side, but would you miss it if you were skimming down the left? maybe. I think this looks better though than scanning longer. Just a thought, not a request.